packages/core/src/components/axes/ruler.ts

```diff
@@ -24,10 +24,17 @@ export class Ruler extends Component {
 		domainValue: number;
 		originalData: any;
 	}[];
+	isRulerEnabled =  Tools.getProperty(
+		this.model.getOptions(),
+		"ruler",
+		"enabled"
+	);
 
 	render() {
 		this.drawBackdrop();
-		this.addBackdropEventListeners();
+		if (this.isRulerEnabled) {
```

Comment:
what if ruler was enabled and now isn't?

Reply:
backdrop event listener will be remove first and then be created is if needed